mlflow/gateway/config.py

```diff
@@ -232,7 +233,8 @@ def validate_mistral_api_key(cls, value):
     Provider.ANTHROPIC: AnthropicConfig,
     Provider.AI21LABS: AI21LabsConfig,
     Provider.MOSAICML: MosaicMLConfig,
-    Provider.BEDROCK: AWSBedrockConfig,
+    Provider.BEDROCK: AmazonBedrockConfig,
+    Provider.AMAZON_BEDROCK: AmazonBedrockConfig,
```

Comment:
To double confirm, the deployments client only supports list and get endpoint. The create endpoint and update  are not implemented, right?
Because the backend change is not done yet.

Reply:
`DatabricksDeploymentClient` supports endpoint creation and update:

https://mlflow.org/docs/latest/python_api/mlflow.deployments.html#mlflow.deployments.DatabricksDeploymentClient

Reply:
As shown in this doc https://docs.google.com/document/d/1ZohOYtfs9JakaZmoWn05UqDuvdJgdCtD2HNOOaw9E44/edit
First step is to update client to accept both `aws-bedrock` and `amazon-bedrock` in the create/get/list/update response, which is already handled by mlflow deployments client.
Second step is to update API to handle both `aws-bedrock` and `amazon-bedrock` in the create/update request.
Third step is to update the client side to use `amazon-bedrock` in the create/update request.

Is this PR only to update the doc and OSS server side and there will be a followup PR to update the create/update on the client side? To be more specific, it's the `aws_bedrock_config` field and `provider` field in the `external_model` object.

Reply:
OSS doesn't support endpoint creation or update.